sotodlib/wiregrid/calibrate.py

```diff
@@ -0,0 +1,407 @@
+#!/usr/bin/python3
+
+# import modules
+# python standard
+import os
+import sys
+import scipy
+from scipy import odr
+import numpy as np
+from iminuit import cost, Minuit
+from matplotlib.patches import Ellipse, Circle
+
+# SO specific
+from sotodlib import core
+from sotodlib.core import Context
+from sotodlib.io import hk_utils
+import sotodlib.io.load_smurf as ls
+from sotodlib.io.load_smurf import Observations
+
+wg_degpercount = 360/52000
+
+def get_det_angle(aman, wg_ctx, wg_offset, stopped_time):
+    """
+    Calibrate TODs and get det_angle_raw by using the wire grid.
+    Before using this code, your axis manager must have two field implemented by HWP, i.e., demodQ and demodU
+
+    Parameters:
+    - aman : AxisManager
```

Comment:
I don't think this docstring format ends up being parsed correctly by sphinx. Worth checking.

Reply:
Are there any sample scripts in sotodlib? I see some scripts have explanations, but others don't. I am not sure which one I should follow.

Reply:
All of the docstrings in here (as one of many examples) are formatted correctly: https://github.com/simonsobs/sotodlib/blob/master/sotodlib/tod_ops/flags.py

I also typically test the docs build myself by running:
```$ virtualenv test
$ source test/bin/activate
$ pip install -r docs/requirements.txt
$ cd docs
$ make html```

That will check that the build works correctly (in addition to the automated github checks) and if it runs successfully will create a new directory called `_build` in the `docs` directory in sotodlib full of html files, you can open those in a browser to see how the formatting looks.

Reply:
Thanks. I'll try it soon.